### PR TITLE
feat: add rochi usdc pool

### PR DESCRIPTION
**Pull request by ChefBingbong**
### Summary
🌾🚀💰

This pull request adds a new farm and a new token for the RociFi project on Ethereum. The farm is for the USDC-ROCI liquidity pool and the token is `ROCI`, defined in `packages/tokens/src/1.ts`.

> _Adding `ROCI` farm_
> _High fee for liquidity_
> _Autumn harvest time_

### Walkthrough
*  Add support for the RociFi project, a decentralized finance platform that offers high returns and low risks for investors ([link](https://github.com/pancakeswap/pancake-frontend/pull/7951/files?diff=unified&w=0#diff-58ebc40aef86d0f8345b23128ffa10d36fa8cd1785c0c6cdc40ea642dae73eb8R282-R288), [link](https://github.com/pancakeswap/pancake-frontend/pull/7951/files?diff=unified&w=0#diff-5f81da2a1b12c306423ba81c127abdc5f541a950a6079e7b4c1d65de467a214eR292-R299))
  * Add a new farm for the USDC-ROCI liquidity pool on Ethereum in `packages/farms/constants/1.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/7951/files?diff=unified&w=0#diff-58ebc40aef86d0f8345b23128ffa10d36fa8cd1785c0c6cdc40ea642dae73eb8R282-R288))
  * Add a new token definition for the ROCI token in `packages/tokens/src/1.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/7951/files?diff=unified&w=0#diff-5f81da2a1b12c306423ba81c127abdc5f541a950a6079e7b4c1d65de467a214eR292-R299))

### CODE REVIEW

File: packages/farms/constants/1.ts [hunk 1]
```diff
@@ -279,6 +279,13 @@ export const farmsV3 = defineFarmV3Configs([
     token1: ethereumTokens.stg,
     feeAmount: FeeAmount.MEDIUM,
   },
+  {
+    pid: 41,
```
Comment (chefjackson) [inline packages/farms/constants/1.ts:283]: let's move it in front of pid 40. The order matters in the farm config

Comment (ChefBingbong) [reply packages/farms/constants/1.ts:283]: sure

File: packages/tokens/src/1.ts [hunk 2]
```diff
@@ -289,4 +289,12 @@ export const ethereumTokens = {
     'PayPal USD',
     'https://www.paypal.com/pyusd',
   ),
+  rochi: new ERC20Token(
```
Comment (chefjackson) [inline packages/tokens/src/1.ts:292]: should be `roci`?

Comment (ChefBingbong) [reply packages/tokens/src/1.ts]: ohh ok changed there

**Review by Chef-Yogi — DISMISSED** [review]

**Review by Chef-Yogi — APPROVED** [review]

**Pull request merged**